README.markdown

```diff
@@ -77,7 +77,7 @@ Synopsis
 
                 -- or connect to a unix domain socket file listened
                 -- by a redis server:
-                --     local ok, err = red:connect("unix:/path/to/redis.sock")
+                --     local ok, err = red:connect("unix:///path/to/redis.sock")
```

Comment:
Why change the unix socket path?

Reply:
I wrote to avoid confusion. 

`red:connect("unix:///var/run/redis/redis-server.sock")`

This is the way the link works. I am using it like this now.

Reply:
@osmanmakal
Unlike `http://`, you don't need double slash after `unix:`. Use `unix:/var/run/redis/redis-server.sock` is enough, so the document doesn't need to be changed.
I guess `unix:///var/run/redis/redis-server.sock` works for you because `///var/run/redis/redis-server.sock` and `/var/run/redis/redis-server.sock` are actually the same.

Reply:
@spacewander Now I have configured it as I gave an example, the writing in the document did not work. You can test it on local...